PENDING.md

```diff
@@ -37,10 +33,18 @@ BREAKING CHANGES
   * [\#3064](https://github.com/cosmos/cosmos-sdk/issues/3064) Sanitize `sdk.Coin` denom. Coins denoms are now case insensitive, i.e. 100fooToken equals to 100FOOTOKEN.
   * [\#3195](https://github.com/cosmos/cosmos-sdk/issues/3195) Allows custom configuration for syncable strategy
   * [\#3242](https://github.com/cosmos/cosmos-sdk/issues/3242) Fix infinite gas
+<<<<<<< HEAD
```

Comment:
@jackzampolin this doesn't look intentional - merge artifact?

Reply:
can we take care of this in the release PR? 😬 sorry about this